.github/workflows/baselines/test_pax_mgmn_metrics.py

```diff
@@ -24,7 +24,7 @@ def test_loss(baseline_filename):
     with open(baseline_filepath, "r") as baseline_file:
         end_step = json.load(baseline_file)["end_step"]
         loss_actual = test_utils.read_tb_tag(event_file, loss_summary_name)
-        assert loss_actual[end_step] < 1.6e-6, f"Loss at final step: {loss_actual[end_step]}, Expected loss < 1.6e-6"
+        assert loss_actual[end_step] < 1.75e-6, f"Loss at final step: {loss_actual[end_step]}, Expected loss < 1.75e-6"
```

Comment:
What do you think about a check on both ends to guard against -inf/negative xent loss, both of which indicating something funny is going on?

```suggestion
        assert 0 <= loss_actual[end_step] < 1.75e-6, f"Loss at final step: {loss_actual[end_step]}, Expected 0 <= loss < 1.75e-6"
```